engine/overworld/sprite_collisions.asm

```diff
@@ -80,7 +80,7 @@ DetectCollisionBetweenSprites:
 	and $f0
 	or c
 
-	ldh [hFF90], a ; store Y coordinate adjusted for direction of movement
+	ldh [hSpriteCollisionsTempYStore], a ; store Y coordinate adjusted for direction of movement
```

Comment:
Delete comments when labels make them obsolete.
```suggestion
	ldh [hSpriteCollisionsTempYStore], a
```

Reply:
these y coord and x coord stores are used for 2 purposes
- storing y or x coord adjusted for direction of movement
- storing the distance between the two sprites' adjusted y or x values

so I figured these comments would help distinguish, since `wSpriteCollisionsTempYCoordOrDistanceBetweenSpriteAdjustedYValue`
would be a bit much...hence why I named it hSpriteCollisionsTempYStore` initially, due to multiple purposes, all being dealing with Y values

Reply:
Maybe I should make them into two?
hSpriteCollisionsTempYCoord::
hSpriteCollisionsDistanceBetweenAdjustedYValues:: db

Reply:
I posted some more label suggestions after reading the file more carefully. Thanks for your patience with all the feedback.